checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallConsistencyAnalyzer.java

```diff
@@ -513,9 +520,13 @@ private boolean shouldTrackInvocationResult(Set<Obligation> obligations, Node no
    *     field or a definitely non-owning pointer
    */
   private boolean returnTypeIsMustCallAliasWithUntrackable(MethodInvocationNode node) {
-    Node mustCallAliasArgument = getMustCallAliasArgumentNode(node);
-    return mustCallAliasArgument instanceof FieldAccessNode
-        || mustCallAliasArgument instanceof ThisNode;
+    List<Node> mustCallAliasArguments = getMustCallAliasArgumentNodes(node);
+    return !mustCallAliasArguments.isEmpty()
+        && mustCallAliasArguments.stream()
+            .allMatch(
+                mustCallAliasArgument ->
+                    mustCallAliasArgument instanceof FieldAccessNode
+                        || mustCallAliasArgument instanceof ThisNode);
```

Comment:
In the Checker Framework, the preferred style is to use a loop rather than a stream. Can you rewrite this to use a loop rather than a stream?

Reply:
https://github.com/typetools/checker-framework/pull/4808/commits/cf5ae953043084ab03b3f20aeec23136e636707b